addons/mail/static/src/core/common/message.js

```diff
@@ -411,12 +411,19 @@ export class Message extends Component {
         if (ev.target.tagName === "A") {
             if (model && id) {
                 ev.preventDefault();
-                this.env.services.action.doAction({
+                await this.env.services.action.doAction({
                     type: "ir.actions.act_window",
                     res_model: model,
                     views: [[false, "form"]],
                     res_id: id,
                 });
+                if (!this.env.isSmall) {
+                    this.store.ChatWindow.insert({
+                        autofocus: 0,
+                        folded: false,
+                        thread: this.props.thread,
```

Comment:
I was thinking this should call `_openChatWindow` instead (with an extra param to avoid auto-focus).
Because manually inserting the chat window would fail to update the state on thread or to notify the sever of new state (cross-tab sync of chat window in particular).